### PR TITLE
Nearby View: Add distance to cards

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
**Description:**
We get this lovely distance info from OTP but haven't made us of it! Now we show this distance info on the top of all the cards and it helps the user get a picture of where all these things around them actually are.

All types are correctly updated and all colors are a11y checked.

Need a French string/translations.

**PR Checklist:**
- [x] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [x] Are all languages supported (Internationalization/Localization)?
- [x] Are appropriate Typescript types implemented?

### CODE REVIEW

**Review by binh-dam-ibigroup — APPROVED** [review]
Looks good with one visual nice-to-have nitpick.

File: lib/components/viewers/nearby/stop-card-header.tsx [hunk 1]
```diff
@@ -99,6 +100,7 @@ const StopCardHeader = ({
             ))}
           <span>{stopData.name}</span>
         </CardTitle>
+        <DistanceDisplay distance={stopData.distance} />
```
[the remark this answers is not in the document]
Comment (miles-grant-ibigroup) [reply lib/components/viewers/nearby/stop-card-header.tsx:103]: Good catch thanks!

**Review by daniel-heppner-ibigroup — CHANGES_REQUESTED** [review]
This looks very good! I wonder if instead of hiding it for 0 we should say "A few x (feet,steps,meters) away" for things under 5 meters away? 

Sometimes the stop you clicked has a distance >0 (I saw 3 ft), which seems a little awkward.

**Comment by miles-grant-ibigroup** [comment]
> This looks very good! I wonder if instead of hiding it for 0 we should say "A few x (feet,steps,meters) away" for things under 5 meters away?
> 
> Sometimes the stop you clicked has a distance >0 (I saw 3 ft), which seems a little awkward.

Good call! Fixed

**Review by daniel-heppner-ibigroup — APPROVED** [review]
LGTM! Thanks for the change

**Pull request merged**